core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -68,6 +68,17 @@ class SparkException(
 }
 
 object SparkException {
+  def internalError(msg: String, context: Array[QueryContext]): SparkException = {
+    val errorClass = "INTERNAL_ERROR"
+    val messageParameters = Map("message" -> msg)
+    new SparkException(
+      message = SparkThrowableHelper.getMessage(errorClass, messageParameters),
```

Comment:
shall we change other `internalError` overloads to load template from json file?

Reply:
@cloud-fan Could you clarify this `to load template from json file`, please. All `internalError` load templates from `error-classes.json`, as I can see `internalError` invoke `SparkThrowableHelper.getMessage()` eventually.